packages/sheets-formula/src/locale/function-list/cube/vi-VN.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default {
+    CUBEKPIMEMBER: {
+        description: 'Trả về các thuộc tính của Chỉ số Hiệu suất Chính (KPI) và hiển thị tên KPI trong ô. KPI là một thước đo có thể đo lường để theo dõi hiệu suất của đơn vị, như tổng lợi nhuận hàng tháng hoặc sự điều chỉnh của nhân viên hàng quý.',
+        abstract: 'Trả về các thuộc tính của Chỉ số Hiệu suất Chính (KPI) và hiển thị tên KPI trong ô. KPI là một thước đo có thể đo lường để theo dõi hiệu suất của đơn vị, như tổng lợi nhuận hàng tháng hoặc sự điều chỉnh của nhân viên hàng quý.',
+        links: [
+            {
+                title: 'Hướng dẫn',
+                url: 'https://support.microsoft.com/zh-cn/office/cubekpimember-%E5%87%BD%E6%95%B0-744608bf-2c62-42cd-b67a-a56109f4b03b',
```

Comment:
All formula translations should be available at `https://support.microsoft.com/vi-vn/office/xxxxx` and `https://support.microsoft.com/zh-tw/office/xxxxx`.